app/Resources/views/interview/reschedule_email.html.twig

```diff
@@ -0,0 +1,13 @@
+<p> Hei, </p>
+
+<p>{{ interview.getUser.getFullName }} har bedt om nytt intervju.</p>
+<p>Tidligere intervjutidspunkt var: {{ interview.getScheduled|date("d.m.y - G:i") }}</p>
+<p>
+    For å sette opp nytt intervju, gå til
+    <a href=https://vektorprogrammet.no{{ path('interview_schedule', { 'id': application.id }) }}>
```

Comment:
I stedet for
```
href=https://vektorprogrammet.no{{ path('interview_schedule', { 'id': application.id }) }}
```
kan du bruke
```
{{ url('interview_schedule', { 'id': application.id }) }}
```

http://symfony.com/doc/current/reference/twig_reference.html#url